packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -0,0 +1,122 @@
+import { CSSRules } from "@microsoft/fast-jss-manager";
+import { DesignSystem } from "../design-system";
+import { format, toPx } from "@microsoft/fast-jss-utilities";
+import { focusOutlineWidth, outlineWidth } from "./design-system";
+
+export const highContrastSelector: string = "@media (-ms-high-contrast:active)";
+
+export enum HighContrastColor {
+    text = "WindowText",
+    hyperLinks = "LinkText",
+    disabledText = "GrayText !important",
```

Comment:
Should we just put important in all of them? It doesn't feel right to have it in only one, but I was thinking as I was going through that the string appeared a lot. Possibly either add an `important()` function or just include it by default in all HC colors.

Reply:
That was the first thing I did, it didn't work. I am guessing it did not know which one to set !important to. That is why I set it in the const, when it needed it.